<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatesMenu</class>
 <widget class="QWidget" name="TemplatesMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="template_menu">
     <property name="font">
      <font>
       <family>Hack</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_creation">
      <attribute name="title">
       <string>Creation</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="hl_name_template">
           <item>
            <widget class="QLabel" name="lb_name_template">
             <property name="text">
              <string>Name template:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_name_template">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl_project_lang">
           <item>
            <widget class="QLabel" name="lb_project_lang">
             <property name="text">
              <string>Project language:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comb_project_language">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C++</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PHP</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_language_form"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QTreeWidget" name="treeW_project_struct">
             <column>
              <property name="text">
               <string>Project struct:</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pb_create_dir">
               <property name="text">
                <string>Create Dir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_create_file">
               <property name="text">
                <string>Create File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_write_in_file">
               <property name="text">
                <string>Write in file</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_unselect">
               <property name="text">
                <string>Unselect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_rename_file">
               <property name="text">
                <string>Rename</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_delete">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pb_reset">
             <property name="styleSheet">
              <string notr="true">padding: 0 13px;
outline: none;
border: 1px solid transparent;  
border-radius: 3px;
height: 32px;
line-height: 32px;
font-size: 14px;
font-weight: 500;
text-decoration: none;
color: #fff;
background-color: red;</string>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_confirm">
             <property name="styleSheet">
              <string notr="true">padding: 0 13px;
outline: none;
border: 1px solid transparent;  
border-radius: 3px;
height: 32px;
line-height: 32px;
font-size: 14px;
font-weight: 500;
text-decoration: none;
color: #fff;
background-color: green;</string>
             </property>
             <property name="text">
              <string>Accept</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_control">
      <attribute name="title">
       <string>Control</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_path_templates">
           <property name="text">
            <string>default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_set_defauilt_path">
           <property name="text">
            <string>Set default path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_select_path">
           <property name="text">
            <string>select path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
